docs/examples/upscale-expanded-particles.ipynb

```diff
@@ -0,0 +1,203 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Revert downsampled, symmetry expanded particles\n",
+    "\n",
+    "Re-extracting particles after symmetry expansion is wasteful, since it creates a particle stack N times larger than is necessary, where N is the number of symmetry-related positions. However, it is often helpful during particle cleanup to work with smaller images, since most cleanup operations do not require high resolution information.\n",
+    "\n",
+    "This notebook illustrates the process of using a subset of particles (presumably after some junk has been removed) to filter the full-size particle images.\n",
+    "\n",
+    "We presume the following jobs have already been run in cryosparc:\n",
+    "\n",
+    " * an initial consensus refinement\n",
+    " * a symmetry expansion of this consensus refinement. This is the full-size symmetry expansion, in our case Job 31.\n",
+    " * a Downsample job of the consensus refinement\n",
+    " * a symmetry expansion of this Downsample job.\n",
+    " * some number of jobs to curate and filter the downsampled, symmetry-expanded particles. This is the downsampled particle stack, in our case Job 34.\n",
```

Comment:
Would it be helpful to include a screenshot of the (partial) workspace tree which includes these jobs?

Reply:
Added with 8bac4890cee0c118682ddd7daf3fcff735ea7ee3